x-pack/plugins/index_management/public/types.ts

```diff
@@ -41,6 +42,7 @@ export interface StartDependencies {
   fleet?: unknown;
   usageCollection: UsageCollectionSetup;
   management: ManagementSetup;
+  ml: MlPluginStart;
```

Comment:
this should be optional, not always definitely present

Reply:
Updated.